deft.el

```diff
@@ -1705,25 +1705,28 @@ Otherwise, quick create a new file."
 (declare-function org-add-link-type "org")
 (declare-function org-open-file-with-emacs "org")
 
+(defun deft-id-at-point ()
+  (substring (expand-file-name (deft-filename-at-point))
+             (1+ (length (expand-file-name deft-directory))))
+  )
```

Comment:
I prefer to keep all closing parens on the same line (see [this guide](https://github.com/bbatsov/emacs-lisp-style-guide)). Here and below.